nntrainer/models/neuralnet.cpp

```diff
@@ -113,6 +113,11 @@ int NeuralNetwork::loadFromConfig(const std::string &config) {
   return ML_ERROR_NONE;
 }
 
+unsigned int NeuralNetwork::getCurrentEpoch() {
+  ml_logd("[NNTrainer] Current epoch: %d", epoch_idx);
```

Comment:
I wonder, why do we need the log here? It means... everytime we call ```getCurrentEpoch()```, the log appear.
If application developer needs this, then he can generate the log in application. Not inside of NNTrainer.